parts/linux/cloud-init/artifacts/sshd_config_2004_fips

```diff
@@ -0,0 +1,125 @@
+#	$OpenBSD: sshd_config,v 1.101 2017/03/14 07:19:07 djm Exp $
```

Comment:
It's copied from "parts/linux/cloud-init/artifacts/sshd_config_1804_fips". Not sure if it's the correct config.

Reply:
we don't need special sshd_config file on 20.04 fips as 18.04 fips requires. So this file can be removed.